soc/arm/st_stm32/stm32f4/Kconfig.series

```diff
@@ -13,5 +13,15 @@ config SOC_SERIES_STM32F4X
 	select HAS_STM32CUBE
 	select CPU_HAS_ARM_MPU
 	select HAS_SWO
+	select HAS_PM
 	help
 	  Enable support for STM32F4 MCU series
+
+if SOC_SERIES_STM32F4X
+
+config PM
+	select COUNTER
+	select COUNTER_RTC_STM32_SUBSECONDS
+	depends on !SOC_SERIES_STM32F1X
```

Comment:
This shouldn't be required

Reply:
Can you say why? Power management depends on using RTC subseconds. Every application that uses `CONFIG_PM=y` would need to set the config `COUNTER_RTC_STM32_SUBSECONDS` manually anyway if I'm not missing something.

Reply:
I've added it because the STM32FX requires `COUNTER_RTC_STM32_SUBSECONDS`. STM32F1x doesn't support the subseconds. The `COUNTER_RTC_STM32_SUBSECONDS` config includes that dependency, but `select` kind of bypass it https://docs.zephyrproject.org/latest/build/kconfig/tips.html#select-pitfalls.

Reply:
But the config is only for SMT32F4 chips.... You are right. I'm correcting this.

Reply:
Done. Sorry for the brain lag :-)